models/query.js

```diff
@@ -1,9 +1,6 @@
 // Represents a request for a subset of objects (reports, incidents, etc.; see subclasses).
 // For example, a query might be 'All reports with status = relevant and keyword foo'
 
-var database = require('../lib/database');
-var mongoose = database.mongoose;
-
```

Comment:
Looks like database and mongoose were never previously used here?

Reply:
Yes.